Sources/NIO/EventLoopFuture.swift

```diff
@@ -283,6 +283,53 @@ public struct EventLoopPromise<T> {
 /// * Using `EventLoopFuture<T>.async` or a similar wrapper function.
 /// * Using `.then()` on an existing future to create a new future for the next step in a series of operations.
 /// * Initializing an `EventLoopFuture` that already has a value or an error
+///
+/// ### Threading and Futures
+///
+/// One of the major performance advantages of NIO over something like Node.js or Python’s asyncio is that NIO will
+/// by default run multiple event loops at once, on different threads. As most network protocols do not require
+/// blocking operation, at least in their low level implementations, this provides enormous speed ups on machines
```

Comment:
I am not a native speaker but I think it should be either `speed-ups` or `speedups`.

Reply:
addressed